embulk-core/src/main/java/org/embulk/deps/EmbulkSelfContainedJarFiles.java

```diff
@@ -86,9 +94,13 @@ static Resource getSingleResource(final String targetResourceName) {
         return resourceToReturn;
     }
 
-    static Collection<Resource> getMultipleResources(final String targetResourceName) {
+    static Collection<Resource> getMultipleResources(final String targetResourceName, final String category) {
+        if (category == null) {
+            return null;
```

Comment:
Let's return an empty list here. Because in `DependencyClassLoader#findResource` we iterate over the returned value of this method.